web-api-db-ctrl/src/elastic.rs

```diff
@@ -168,28 +162,6 @@ async fn get_opt_tenant_mapping(elastic: &Client) -> Result<Option<Value>, Error
     }
 }
 
-#[instrument(skip(elastic))]
-async fn create_alias_index_to(elastic: &Client, original_index: &str) -> Result<(), Error> {
```

Comment:
We don't need that anymore it was a workaround for not having the feature of being able to give a tenant a index with an arbitrary name.